pkg/util/roundtrippers/retrying.go

```diff
@@ -95,16 +95,17 @@ func NewRetryingRoundTripper(log *logrus.Entry, location string, privateEndpoint
 }
 
 func (rt *RetryingRoundTripper) RoundTrip(req *http.Request) (resp *http.Response, err error) {
+	cancel := make(chan struct{})
```

Comment:
this change doesn't look right - why are you making it?

Reply:
Reverted.